File: KosmosToolkit/src/main/java/cos/mos/toolkit/ui/toast/UToast.java
package cos.mos.toolkit.ui.toast;

import android.widget.Toast;

import cos.mos.toolkit.init.k.KApp;


/**
 * @Description: <p>
 * @Author: Kosmos
 * @Date: 2019.03.21 11:34
 * @Email: KosmoSakura@gmail.com
 */
public class UToast {
    public static void show(String str) {
        Toast.makeText(KApp.instance(), str, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/cos/mos/utils/ukotlin/UToast.kt
package cos.mos.utils.ukotlin

import android.widget.Toast
import cos.mos.toolkit.init.KApp

/**
 * @Description:
 * <p>
 * @Author: Kosmos
 * @Date: 2019.04.23 13:21
 * @Email: KosmoSakura@gmail.com
 */
object UToast {
    @JvmStatic
    fun show(str: String) {
        Toast.makeText(KApp.instance(), str, Toast.LENGTH_SHORT).show()
    }
}
